R/options.R

```diff
@@ -0,0 +1,56 @@
+#' Package options
+#'
+#' These options can be set via [options()] and queried via [getOption()].
+#' For this, add a `styler.` prefix (the package name and a dot) to the option name.
+#' Example: for an option `foo`, use `options(styler.foo = value)` to set it
+#' and `getOption("styler.foo")` to retrieve the current value.
+#' An option value of `NULL` means that the default is used.
+#'
+#' @usage NULL
+#' @format NULL
+#'
+#' @name styler_options
+#'
+#' @examplesIf FALSE
+#' getOption("styler.ignore_alignment")
+#' options(
+#'   styler.ignore_alignment = TRUE,
+#'   styler.quiet = TRUE
+#' )
+#' getOption("styler.ignore_alignment")
+#'
+#' @section Options for the styler package:
+#'
+#' - `styler.addins_style_transformer`: character.
+#'   The name of the style transformer to use in the addins.
+#'   (default: `"styler::tidyverse_style()"`)
+#'
+#' - `styler.cache_name`: character.
+#'   The name of the styler cache to use.
+#'   (default: current styler package version)
```

Comment:
We could actually put the version here, since roxygen comments can now contain R code to evaluate.